docker/Dockerfile.release

```diff
@@ -1,69 +1,10 @@
-# This dockerfile is used for quay.io images
-FROM ubuntu:bionic as builder
+ARG DISTRO=focal
+FROM ubuntu:$DISTRO
 
-ARG bcc_ref="v0.12.0"
-ARG LLVM_VERSION="8"
-ENV LLVM_VERSION=$LLVM_VERSION
+ARG build_dir=build-embedded
 
-RUN apt-get update && apt-get install -y curl gnupg &&\
-    llvmRepository="\n\
-deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main\n\
-deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main\n\
-deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-${LLVM_VERSION} main\n\
-deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic-${LLVM_VERSION} main\n" &&\
-    echo $llvmRepository >> /etc/apt/sources.list && \
-    curl -L https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+# Run security updates
+RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
```

Comment:
do we need to do this? Won't ubuntu keep  their images up to date?

Reply:
When I ran this against Ubuntu bionic it dropped the vulnerability count from 7 to 4, according to the quay scanner, so I guess there is some lag on security updates in upstream images 

I also tried Debian buster but it had a high severity one, so I switched to focal. It might be that focal is just so new that there’s no vulns yet.

In any case it doesn’t matter tremendously, the only thing bpftrace really needs I think is glibc.